Add references to RTP taxonomy draft (#14)
murillo128/codec-agnostic-rtp-payload-format: codec-agnostic-rtp-payload-format.md

The objective of this spec is to create a generic RTP packetization format that can be used with any video codec or encrypted content and that allows SFUs to perform layer selection without requiring access to the codec payload.

> Media paquetization and depaquetization

Reviewer: ```suggestion
Media packetization and depacketization
```

=======================

> As per {{RFC7656}} the generic packetizer will define a Media Packetizer allows to transfor a single Encoded Stream into one or several RTP packets  

Reviewer: ```suggestion
As per {{RFC7656}} the generic packetizer will define a Media Packetizer that transforms a single Encoded Stream into one or several RTP packets.
```

>  which can be therefore be processed by a Media Depacketizer to reconstruct the Encoded Stream.

Reviewer: ```suggestion
These RTP packets are sent over the wire to a Media Depacketizer that will reconstruct the Encoded Stream.
```

           +----------------------+
           |    Media Encryptor   |
           +----------------------+
                      |
                Encrypted Stream    +------------+
                      V             |            V
           +----------------------+ | +----------------------+
           |   Media Packetizer   | | | RTP-Based Redundancy |
           +----------------------+ | +----------------------+
                      |             |            |
                      +-------------+  Redundancy RTP Stream
               Source RTP Stream                 |
                      V                          V
           +----------------------+   +----------------------+
           |  RTP-Based Security  |   |  RTP-Based Security  |
           +----------------------+   +----------------------+
                      |                          |
              Secured RTP Stream   Secured Redundancy RTP Stream
                      V                          V
           +----------------------+   +----------------------+
           |   Media Transport    |   |   Media Transport    |
           +----------------------+   +----------------------+

>              Figure 1: Sender Side Concepts in the Media Chain with Encryption 

Reviewer: ```suggestion
             Figure 1: Sender Side Concepts in the Media Chain with Application-level Media Encryption 
```

             
```
 
This generic packetization does not change how the mapping between one or several encoded or dependant streams are mapped to the RTP streams or how the synchronization sources(s) (SSRC) are assigned. 

> The generic packetizer only supports SRST modes when Scalale Video Coding (SVC) is in use.

Reviewer: ```suggestion
The generic packetizer only supports Single RTP stream on a Single media Transport (SRST) when Scalale Video Coding (SVC) is in use.
```
